sections/memory_safety_in_rust.tex

```diff
@@ -35,6 +32,29 @@ \section{Memory Safety in Rust}
 
 \end{frame}
 
+
+%%%
+
+\begin{frame}[fragile]{Ergonomic scaffolding with \texttt{unwrap()}}
```

Comment:
Allowing to draft a program quickly without writing all the error handling code.